### DISCUSSION TITLE
Uid's

### DISCUSSION THREAD

**Discussion by pfisterjonas** [django-commons/django-unicorn#533] [Q&A]
Dear all,

I ran into some issues using UUIDs with django-unicorn. 
I always use uuid4 as primary keys, but for some reason everytime I use them in django-unicorn, the response values in the template get doubled. 
I think there is an issue with the serialization of uuid-fields. The work around I have found is to pass the uuid in as string into a Charfield. Then the issue does not occur.  Anyone else have a similar experience?

Best regards,

J

**Comment by adamghill** [comment]
Can you show an example screenshot of what you mean by "the response values in the template get doubled"?  I use `orjson` to serialize data and it should handle `uuid` as far as I know. I have a few tests around `uuid`s as well, e.g. https://github.com/adamghill/django-unicorn/blob/de4c66376ea5f5f21f51ef5e4ae5b84dede3e378/tests/serializer/test_model_value.py#L24. But, I definitely might be something missing somewhere. :)
